README.md

```diff
@@ -69,6 +70,19 @@ To produce a signed URL, you must enable secure URLs on your source and then pro
 
 ```
 
+## Disabled Path Encoding
+
+Path encoding is enabled by default. It can be toggled off by setting `disable_path_encoding` to `True`:
+
+```python
+>>> from imgix import UrlBuilder
+>>> ub = UrlBuilder("demo.imgix.net", disable_path_encoding=True)
+>>> ub.create_url("file+with%20some+crazy?things.jpg", {'w': 100, 'h': 100})  
+'https://demo.imgix.net/file+with%20some+crazy?things.jpg?h=100&w=100'
+```
+
+Normally this would output a src of `https://demo.imgix.net/file+with%20some+crazy?things.jpg?h=100&w=100`, but since path encoding is disabled, it will output a src of `https://demo.imgix.net/file+with%20some+crazy?things.jpg?h=100&w=100`.
```

Comment:
Same as the previous comment, I think an example with 1) invalid path characters and 2) that shows a clear difference before/after encoding is disabled would serve us better here.

```suggestion
Normally this would output a source URL like `http://sdk-test.imgix.net/%20%3C%3E%5B%5D%7B%7D%7C/%5E%.jpg?h=100&2=100`, but since path encoding is disabled, it will output a source URL like [http://sdk-test.imgix.net/ <>[]{}|^%.jpg](http://sdk-test.imgix.net/%20%3C%3E%5B%5D%7B%7D%7C/%5E%.jpg).
```